src/main/java/com/xebialabs/deployit/ci/DeployitNotifier.java

```diff
@@ -365,6 +375,11 @@ public FormValidation doCheckApplication(@QueryParameter String credential, @Que
         }
 
         public FormValidation doReloadTypes(@QueryParameter String credential, @AncestorInPath AbstractProject project) {
```

Comment:
Side effect, therefore needs to be limited to POST requests. See https://jenkins.io/doc/developer/security/form-validation/#protecting-from-csrf

Reply:
added
```
@POST
public FormValidation doReloadTypes(@QueryParameter String credential, @AncestorInPath AbstractProject project) {
```